ui/js/component/file-selector.js

```diff
@@ -47,16 +52,31 @@ class FileSelector extends React.PureComponent {
       <div className="file-selector">
         <button
           type="button"
-          className="file-selector__choose-button"
+          className="button-block button-alt file-selector__choose-button"
           onClick={() => this.handleButtonClick()}
         >
-          {this.props.type == "file"
-            ? __("Choose File")
-            : __("Choose Directory")}
+          <span className="button__content">
+            <span className="button-label">
+              {this.props.type == "file"
+                ? __("Choose File")
+                : __("Choose Directory")}
+            </span>
+          </span>
         </button>
         {" "}
         <span className="file-selector__path">
-          {this.state.path ? this.state.path : __("No File Chosen")}
+          <input
+            className="input-copyable"
```

Comment:
I understand the text is selected to make this copyable, but it also gives the user an implication that they can type in this field. Can we somehow express/provide both?

Reply:
I don't think the user should be allowed to type a directory / file, it's better letting the file dialog handle this to ensure that the file / directory exists, considering the current validaton system :stuck_out_tongue:

Reply:
I was unclear; I agree users should not be allowed to type in this field.

Is there a style we can apply that indicates BOTH copyable AND readonly?